rules/linux/lateral_movement_telnet_network_activity_internal.toml

```diff
@@ -34,8 +34,8 @@ query = '''
 sequence by process.entity_id
   [process where process.name == "telnet" and event.type == "start"]
   [network where process.name == "telnet" and
-    destination.ip in ("10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16", "FE80::/10") and
-    destination.ip not in ("127.0.0.0/8", "::1/128")]
+    cidrmatch(destination.ip, "10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16", "FE80::/10") and
+    not cidrmatch(destination.ip, "127.0.0.0/8", "::1/128")]
```

Comment:
can you or @bm11100 add one screenshot for `cidrmatch` (lowercase) working in the ES API?

Reply:
added